tracer/test/test-applications/integrations/Samples.GraphQL4/StarWarsExtensions/StarWarsSubscription.cs

```diff
@@ -1,11 +1,12 @@
-#if !GRAPHQL_5_0
 using System;
 using System.Collections.Generic;
 using System.Reactive.Linq;
 using System.Reactive.Subjects;
 using GraphQL;
 using GraphQL.Resolvers;
+#if !GRAPHQL_5_0 && !GRAPHQL_7_0
```

Comment:
really good work here handling all versions. you can also have _partial_ classes dividing the same class by preprocessor directives for example [here](https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Core.cs) and https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs